app/validations/file-cleanup.js

```diff
@@ -0,0 +1,12 @@
+import {
+  validateNumber
+} from 'ember-changeset-validations/validators';
+
+
+export default {
+  filesToKeep: validateNumber({
+    gte: 2,
+    lte: 50,
+    message: "Files to keep must be between 2 and 50"
```

Comment:
rename `Files to keep must be between 2 and 50` to `value must be between 2 and 50`